source/dub/project.d

```diff
@@ -1421,6 +1421,7 @@ private string[] processVarsWithGlob(Project, Package)(string var, in Project pr
 		{
 		case '*', '?', '[', '{': break loop;
 		case '/': sepIdx = i; goto default;
```

Comment:
Does `/` work as a separator on Windows?
If not, we need `version(Posix)` for this case statement.

Reply:
Yes, but it may depend upon UNC library.

https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats#canonicalize-separators

Reply:
In Windows, both `/` and `\` function as path separators.
In fact, it is already converted to `\` by NativePath.toNativeString in the previous processVars, but it is better to work more robustly.